solrsearch-plugins/docs/SolrSearch-batchsink.md

```diff
@@ -0,0 +1,71 @@
+# SolrSearch Batch Sink
+
+
+Description
+-----------
+The SolrSearch Batch Sink takes the structured record from the input source and indexes it into the Solr server using
+the collection and idField specified by the user.
+
+The incoming fields from the previous stage in pipelines are mapped to Solr fields. Also, user is able to specify the
+mode of the Solr to connect to. For example, Single Node Solr or SolrCloud.
+
+Use Case
+--------
+SolrSearch Batch Sink is used to write data to the Solr server. For example, the plugin can be used in conjuction
```

Comment:
Should we talk about SolrSearch as Solr Search. Don't need to use the name of the plugin in documentation.